ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-release-ocm-2.8.yaml

```diff
@@ -73,9 +71,6 @@ resources:
     requests:
       cpu: 100m
       memory: 200Mi
-test_binary_build_commands: |
```

Comment:
@danmanor I removed those lines to align with other releases. Not sure if it's required